STAR-Delegation/CSR-template/test/GOOD_full.json

```diff
@@ -17,8 +17,8 @@
       "DNS": [
         "cache.ndc.ido.example"
       ],
-      "IP": [
-        "*"
+      "URI": [
+        "https://cache.ndc.ido.example"
```

Comment:
URIs are never used for HTTP resources (as far as I know). Quoting RFC 6125: 
> Therefore this
      document discusses Uniform Resource Identifiers [URI] only as a
      way to communicate a DNS domain name (via the URI "host" component
      or its equivalent), not as a way to communicate other aspects of a
      service such as a specific resource (via the URI "path" component)
      or parameters (via the URI "query" component).
      
A better example would be (from the same RFC): `sip:voice.example.edu`.

Reply:
cool, will update